ci/install-check.sh

```diff
@@ -17,7 +17,9 @@ TEMP="$(mktemp -d -t TEMP.XXXXXXX)" || die "failed to make tmpdir"
 cleanup() { [[ -n "${TEMP:-}" ]] && rm -rf "${TEMP}"; }
 trap cleanup EXIT
 
-TOPLEVEL="$(git -C "$(cd "$(dirname "$0")" >/dev/null || exit 1; pwd)" rev-parse --show-toplevel)" || die 'failed to find TOPLEVEL'
+realpath() { python -c "import os,sys; print os.path.realpath(sys.argv[1])" "$1"; }
```

Comment:
So this assumes python directs to python2. Probably fine for now, just want to point out.

Reply:
ugh, ok you're right. i'll fix it

Reply:
fixed in #199